package/CHANGELOG

```diff
@@ -34,6 +34,8 @@ Fixes
  * Fix deploy action to use the correct version of the pypi upload action.
 
 Enhancements
+* Added a tqdm progress bar for `MDAnalysis.analysis.pca.PCA.transform()`
```

Comment:
```suggestion
 * Added a tqdm progress bar for `MDAnalysis.analysis.pca.PCA.transform()`
```
Very minor but it can make the next person have slightly different formatting, etc...

Reply:
please fix